images/fedora/f37/Containerfile

```diff
@@ -13,6 +13,8 @@ COPY README.md /
 
 RUN sed -i '/tsflags=nodocs/d' /etc/dnf/dnf.conf
 RUN dnf -y swap coreutils-single coreutils-full
+RUN dnf -y swap glibc-minimal-langpack glibc-all-langpacks
+RUN rm /etc/rpm/macros.image-language.conf
```

Comment:
Typo alert!  Did you try a `podman build ...` with these changes?  :)

The file is `/etc/rpm/macros.image-language-conf`, not `/etc/rpm/macros.image-language.conf`.

Reply:
Fixed in https://github.com/containers/toolbox/pull/1225